app/assets/javascripts/discourse/app/templates/signup.hbs

```diff
@@ -140,7 +142,9 @@
                       @validation={{this.passwordValidation}}
                       id="password-validation"
                     />
-                  {{else if this.passwordInstructions}}
+                  {{else if
+                    this.siteSettings.show_signup_form_password_instructions
+                  }}
                     <span class="more-info" id="password-validation-more-info">
                       {{this.passwordInstructions}}
```

Comment:
wasn't `passwordInstructions` removed?

Reply:
It comes from this:

https://github.com/discourse/discourse/blob/88f1b3b195409d534f86c2b609767528ad6fbc5e/app/assets/javascripts/discourse/app/mixins/password-validation.js#L15-L19

 I hadn’t realized I was overriding it earlier.